hal/src/hal-block_drivers.ads

```diff
@@ -9,13 +9,13 @@ package HAL.Block_Drivers is
 
    function Read
      (This         : in out Block_Driver;
-      Block_Number : Unsigned_32;
```

Comment:
This breaks services/src/utils/file_block_drivers.ads, services/src/filesystems/partitions.adb and examples/sdcard/adafatfs/fat_filesystem.adb.